### ISSUE TITLE
error messages

### ISSUE THREAD

**Issue by mkudlej** [3scale/3scale_toolbox#66]
There should be error message with explanation if command failed. Also return code should be non-zero in case of failure or error.

**Comment by mikz** [comment]
Do you have an example?

**Comment by mkudlej** [comment]
@mikz Sure:
```
$ 3scale import csv --destination="https://_valid_token_@_valid_url_" --file=import.csv
Available subcommands: 
copy
help
import
update

Usage: 3scale <command> [options]
    -h, --help                       Prints this help
$ echo $?
0
$ rpm -q 3scale_toolbox
3scale_toolbox-0.4.0-1.el7.noarch
```
File ```import.csv``` contains example from https://github.com/3scale/3scale_toolbox#import-from-csv and services were not imported.
I don't know what happened and what's wrong.

**Comment by eguzki** [comment]
seems packaging issue

**Comment by eguzki** [comment]
It was packaging issue not related to this repo.

I keep the issue open to check about

```return code should be non-zero in case of failure or error```

**Comment by eguzki** [comment]
`return code should be non-zero in case of failure or error` implemented in #73 

closing
